Changes directories where various files (i.e. logfiles, templates, config files)
are stored from the privoxy defaults. 
Solaris-specific patch; not suitable for upstream

diff -rupN privoxy-3.0.23-stable-orig/config privoxy-3.0.23-stable/config
--- privoxy-3.0.23-stable-orig/config	2015-06-02 13:27:15.297393314 -0700
+++ privoxy-3.0.23-stable/config	2015-06-02 13:29:14.468549370 -0700
@@ -260,7 +260,7 @@
 #
 #      No trailing "/", please.
 #
-confdir .
+confdir /etc/privoxy 
 #
 #  2.2. templdir
 #  ==============
@@ -345,7 +345,7 @@ confdir .
 #
 #      No trailing "/", please.
 #
-logdir .
+logdir /var/log/privoxy 
 #
 #  2.5. actionsfile
 #  =================
@@ -478,7 +478,7 @@ filterfile user.filter      # User custo
 #      require additional software to do it. For details, please
 #      refer to the documentation for your operating system.
 #
-logfile logfile
+logfile privoxy.log 
 #
 #  2.8. trustfile
 #  ===============
